client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -201,13 +206,15 @@ export const PrescriptionLineEditForm: React.FC<
     const selectedItem = items.find(
       prescriptionItem => prescriptionItem.id === item?.id
     );
-    const newPrescribedQuantity: number =
-      selectedItem?.lines?.find(
-        ({ prescribedQuantity }) =>
-          prescribedQuantity != null && prescribedQuantity > 0
-      )?.prescribedQuantity ?? 0;
+    if (preferences?.editPrescribedQuantityOnPrescription) {
+      const newPrescribedQuantity: number =
+        selectedItem?.lines?.find(
+          ({ prescribedQuantity }) =>
+            prescribedQuantity != null && prescribedQuantity > 0
+        )?.prescribedQuantity ?? 0;
 
-    setPrescribedQuantity(newPrescribedQuantity);
+      setPrescribedQuantity(newPrescribedQuantity);
+    }
```

Comment:
The testers probably won't be able to tell, but we should confirm that the "presribedQuantity" is NOT updated (i.e. left as `null`) when the preference is off.